Update theme description

andersnoren opened the pull request:
Replaces the placeholder theme description in `style.css`, `readme.md` and `readme.txt`.

ianbelanger79 commented on README.md:
```diff
@@ -9,7 +9,7 @@
 ## Description
 
-Default theme for Twenty Twenty
+Our default theme for 2020 is designed to take full advantage of the flexibility of the block editor. Organisations and businesses can create landing pages with dynamic layouts using the group and columns blocks, and the centered content column and fine-tuned typography makes it perfect for traditional blogs as well. Full block editor styles give you a good idea of what the final result will look ...
```
What do we think of changing 

"**Organisations and businesses can create landing pages with dynamic layouts using the group and columns blocks, and the centered content column and fine-tuned typography makes it perfect for traditional blogs as well.**"

to

"**Organizations and businesses have the ability to create dynamic landing pages with endless layouts using the group and column blocks. The centered content column and fine-tuned typography also makes it perfect for traditional blogs.**"

and 

"**Full block editor styles give you a good idea of what the final result will look like before you publish.**"

to 

"**Full block editor styles give you a preview of what the final result will look like, even before you publish.**"

joyously replied:
Or 
"The centered content column and fine-tuned typography make it perfect for landing pages or traditional blogs."

and just remove that other sentence.

andersnoren commented:
@ianbelanger79 Good suggestions! Updated the description accordingly. I also modified the last couple of sentences to make the overall description a bit shorter, and reduced the length of the second to last sentence.

> Our default theme for 2020 is designed to take full advantage of the flexibility of the block editor. Organizations and businesses have the ability to create dynamic landing pages with endless layouts using the group and column blocks. The centered content column and fine-tuned typography also makes it perfect for traditional blogs. Full block editor styles give you a good idea of what your content will look like, even before you publish. You can give your site a personal touch by changing the background colors and the accent color in the Customizer. The colors of all elements on your site are automatically calculated based on the colors you pick, ensuring a high, accessible color contrast for your visitors. 

@carolinan @pattonwebz

carolinan commented:
I have to read this part "Full block editor styles" several times because it was not clear to me if it meant 
"Full width block" or full as in complete/all.  Maybe the word can be removed.

andersnoren commented:
@carolinan Good point. I suppose we don't need to specify "block editor styles" as well, since we include editor styles for both the block editor and the classic editor. I changed the beginning of the sentence from ”Full block editor styles” to ”Complete editor styles”.
